packages/web/src/components/search/DataSearch.js

```diff
@@ -5,45 +5,46 @@ import { withTheme } from 'emotion-theming';
 import {
 	addComponent,
 	removeComponent,
-	watchComponent,
-	updateQuery,
-	setQueryOptions,
-	setQueryListener,
 	setComponentProps,
-	updateComponentProps,
+	setQueryListener,
+	setQueryOptions,
 	setSuggestionsSearchValue,
+	updateComponentProps,
+	updateQuery,
+	watchComponent,
```

Comment:
@ShahAnuj2610 Remove these unnecessary changes, we talked about it.

Reply:
done